ArduPlane/ArduPlane.cpp

```diff
@@ -908,6 +908,50 @@ bool Plane::set_land_descent_rate(float descent_rate)
 #endif
     return false;
 }
+
+// allow scripts to find access prev_WP_loc
+bool Plane::get_previous_location(Location &previous_location)
```

Comment:
I think Pete is right that this functionality can be done in the script without a binding